### PR TITLE
README: Add README.md for eatmemory test

### PR DESCRIPTION
**Pull request by Geet2**
Add README.md for memory/eatmemory.py test

### CODE REVIEW

File: memory/eatmemory.py.data/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+eatmemory is a test case that is designed to consume a specified amount of memory on a system to test the behavior of swap and the system in general when there is little free memory available with it. This can be useful for testing the memory capacity of a system or for stressing the system in order to measure its performance under heavy loads. Based on the size of memory supplied to this test, it allocates memory using the malloc() and then uses on memset() on it to make sure the page fault occurs.
+
+Source : https://github.com/julman99/eatmemory/ (Julio Viera <julio.viera@gmail.com>)
+
+
+Prerequisites
+--------------
+* To run eatmemory, you will need a Linux system with Python 3.x and the avocado-framework package installed.
+
+* By default memory to test is 95% of the system's free memory. If you want to change that you can change the memory_to_test parameter in the yaml file.
```
Comment (Naresh-ibm) [inline memory/eatmemory.py.data/README.md:12]: @Geet2 Please add the yaml parameter here and add the description for them too. like what different values we can pass to this parameter and any other conditions one should be aware off, while passing the value for the parameter.

**Review by Naresh-ibm — CHANGES_REQUESTED** [review]

**Review by Naresh-ibm — APPROVED** [review]
@Geet2 LGTM
Thanks for the changes and PR

**Comment by Naresh-ibm** [comment]
@PraveenPenguin Please ACK, so that I can merge this PR

**Review by PraveenPenguin — APPROVED** [review]
LGTM

**Review by Naresh-ibm — COMMENTED** [review]
@Geet2 Thanks for PR,
@PraveenPenguin Thanks for review.

**Pull request merged**